index.html

```diff
@@ -276,7 +282,7 @@ <h2 class="contact-message">
             ><img src="images/icons/angellist.svg" alt="angellist"
           /></a>
         </div>
-        <div>
+        <div class="email">
```

Comment:
- [ ] great job. Everything looks fine and it's responsive. Please kindly use the right background image on the contact section.
Figma Version
![Screenshot (324)_LI](https://user-images.githubusercontent.com/65298048/186502439-414b0280-3b28-4def-9bc3-dd0164d056dc.jpg)
your version

![Screenshot (323)_LI](https://user-images.githubusercontent.com/65298048/186502467-ff07379a-13e4-497f-96d2-a5e683d043d8.jpg)